opendbc/car/lateral.py

```diff
@@ -1,15 +1,26 @@
+import math
 import numpy as np
 from dataclasses import dataclass
-from opendbc.car import structs
+from opendbc.car import structs, rate_limit, DT_CTRL
+from opendbc.car.vehicle_model import VehicleModel
 
 FRICTION_THRESHOLD = 0.3
 
+# Add extra tolerance for average banked road since safety doesn't have the roll
+AVERAGE_ROAD_ROLL = 0.06  # ~3.4 degrees, 6% superelevation. higher actual roll lowers lateral acceleration
+
+# ISO 11270
+ISO_LATERAL_ACCEL = 3.0  # m/s^2
+ISO_LATERAL_JERK = 5.0  # m/s^3
+
 
 @dataclass
 class AngleSteeringLimits:
   STEER_ANGLE_MAX: float
   ANGLE_RATE_LIMIT_UP: tuple[list[float], list[float]]
   ANGLE_RATE_LIMIT_DOWN: tuple[list[float], list[float]]
+  MAX_LATERAL_ACCEL: float = ISO_LATERAL_ACCEL * .7  # ~2.1m/s^2, 70% ISO limits as default, but ideally this is set manually for comfortable limits?
+  MAX_LATERAL_JERK: float = ISO_LATERAL_JERK * .5  # ~2.5m/s^3, 50% of ISO limits as default, but ideally this is set manually for comfortable limits?
```

Comment:
Where did these come from? It shouldn't be possible for cars that aren't tested with this function yet to be able to use it by default

Reply:
Some conservative values / sensible defaults since `AngleSteeringLimits` seems used by cars that did not use the VM methods. Whether `safety` would allow them or not would be another topic... Should we do `None` instead?